Packs/Jira/Integrations/JiraV2/JiraV2.py

```diff
@@ -741,6 +743,64 @@ def edit_issue_command(issue_id, mirroring=False, headers=None, status=None, tra
     return get_issue(issue_id, headers, is_update=True)
 
 
+def append_to_field_command(issue_id, fieldJson, headers=None, **kwargs):
+    issue = jira_req('GET', f'rest/api/latest/issue/{issue_id}', resp_type='json')
+
+    for field in fieldJson:
+        field_type = __get_field_type(field)
+        if not field_type:
+            return_error(f"field {field} could not be updated.")
+
+        if field_type == 'Field Not Found':
+            return_error(f'Could not identify field {field}. Make sure it was entered with the correct field id.')
+
+        current_data_in_field = issue.get('fields', {}).get(field)
+        new_data = {}
+        if not current_data_in_field:
+            new_data[field] = __create_value_by_type(field_type, fieldJson[field])
+        else:
+            new_data[field] = __add_value_by_type(field_type, current_data_in_field, fieldJson[field])
+
+        url = f'rest/api/latest/issue/{issue_id}/'
+        jira_req('PUT', url, json.dumps({'fields': new_data}))
+
+    return get_issue(issue_id, headers, is_update=True)
+
+
+def get_field_command(issue_id, field, **kwargs):
+    fields=argToList(field)
+    return get_issue(issue_id, customfields=fields, is_update=False)
+
+
+def __get_field_type(field_id):
+    fields = jira_req('GET', 'rest/api/2/field')
+    field_data_filter = filter(lambda x: x.get('id') == field_id, fields.json())
+    try:
+        field_data = next(field_data_filter)
+        return field_data.get('schema', {}).get('type')
+    except StopIteration:
+        return 'Field Not Found'
```

Comment:
Which field? Please add to the error message.

Reply:
It will return the error to the function above which in turn will add the field name. (The previous comment here is exactly this function)